docs/guides/beginnerCLI.rst

```diff
@@ -0,0 +1,276 @@
+
+.. _YAML: https://yaml.org/
+
+Importing Your Library
+----------------------
+
+The next step is to import your music files into the beets library database.
+Because this can involve modifying files and moving them around, data loss is
+always a possibility, so now would be a good time to make sure you have a
+recent backup of all your music. We'll wait.
+
+There are two good ways to bring your existing library into beets. You can
+either: (a) quickly bring all your files with all their current metadata into
+beets' database, or (b) use beets' highly-refined autotagger to find canonical
+metadata for every album you import. Option (a) is really fast, but option (b)
+makes sure all your songs' tags are exactly right from the get-go. The point
+about speed bears repeating: using the autotagger on a large library can take a
+very long time, and it's an interactive process. So set aside a good chunk of
+time if you're going to go that route. For more on the interactive
+tagging process, see :doc:`tagger`.
+
+If you've got time and want to tag all your music right once and for all (option b), do
+this::
+
+    beet import /path/to/my/music
+
+For this command to work, you must input the full path name. 
+
+(Note that by default, this command will *copy music into the directory you
+specified above*. If you want to use your current directory structure, set the
+``import.copy`` config option.) 
+
+A successful import will look as follows:
+
+    .. image:: ../_static/Images/firstImport.png
+        :width: 600
+
+Once imported, follow the prompts in the CLI to tag the music as you see fit. 
+
+To take the fast, un-autotagged path (option a), just say::
+
+    beet import -A /my/huge/mp3/library
+
+Note that you just need to add ``-A`` for "don't autotag" option.
+
+Adding More Music
+-----------------
+
+If you've ripped or... otherwise obtained some new music, you can add it with
+the ``beet import`` command, the same way you imported your library. Like so::
+
+    beet import ~/some_great_album
+
+This will attempt to autotag the new album (interactively) and add it to your
+library. There are, of course, more options for this command---just type ``beet help import`` to see what's available.
+
+Seeing Your Music
+-----------------
+
+If you want to query your music library, the ``beet list`` (shortened to ``beet
+ls``) command is for you. You give it a :doc:`query string </reference/query>`,
+which is formatted something like a Google search, and it gives you a list of
+songs.  Thus::
+
+    beet ls the magnetic fields
+    The Magnetic Fields - Distortion - Three-Way
+    The Magnetic Fields - Distortion - California Girls
+    The Magnetic Fields - Distortion - Old Fools
+    beet ls hissing gronlandic
+    of Montreal - Hissing Fauna, Are You the Destroyer? - Gronlandic Edit
+    beet ls bird
+    The Knife - The Knife - Bird
+    The Mae Shi - Terrorbird - Revelation Six
+    beet ls album:bird
+    The Mae Shi - Terrorbird - Revelation Six
```

Comment:
Perhaps indent or format this as a shell output; the commands and the output look the same, makes it hard to pick out the shell commands.